src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/ServerType.cs

```
@@ -5,9 +5,10 @@ namespace Microsoft.AspNetCore.Server.IntegrationTesting
 {
     public enum ServerType
     {
+        None,
```

What the use case for None?

---

So we can tell if a field was accidentally not set. I want to avoid defaulting to an incorrect value.

---

Nullable?